infrastructure/src/main/java/org/corfudb/infrastructure/ServerContext.java

```diff
@@ -617,19 +572,12 @@ private EventLoopGroup getNewBossGroup() {
      */
     private @Nonnull EventLoopGroup getNewWorkerGroup() {
```

Comment:
maybe name of the method should also be improved. It is a netty io worker group so name should be indicating that ?